include/sys/time.h

```diff
@@ -163,8 +173,27 @@ struct itimerval {
 
 #ifdef _KERNEL
 
+#include <sys/callout.h>
+
 typedef struct proc proc_t;
 
+/* Kernel interval timer. */
+typedef struct {
+  timeval_t kit_next;     /* absolute time of next tick, 0 means inactive */
+  timeval_t kit_interval; /* time between ticks, 0 means non-periodic */
```

Comment:
Instead ticks I would use expire date/time and intervals. IMO the name ticks is too general.

Reply:
I've changed 'ticks' to 'expirations'.